lib/moon_web/components/left_menu.ex

```diff
@@ -15,13 +15,19 @@ defmodule MoonWeb.Components.LeftMenu do
   prop uri, :any
   prop active_page, :any
   prop hide_items, :boolean
+  prop click, :event
 
   def render(assigns) do
     ~F"""
-    <div class={
-      "hidden fixed h-screen lg:flex lg:flex-shrink-0 w-80 flex-col",
-      @theme_name
-    }>
+    <div
+      phx-click={@click}
```

Comment:
:on-click is for surface

Reply:
the same - LiveView event is used to set target to the parent LiveView

Reply:
If someone starts using this component from other component, then automatic behaviour might be unexpected. Let's try to use :on-click